build.gradle

```diff
@@ -23,12 +23,31 @@ repositories {
 	mavenCentral()
 }
 
-
 dependencies {
 //	implementation('org.springframework.boot:spring-boot-starter-data-jpa')
 	implementation('org.springframework.boot:spring-boot-starter-websocket')
 //	implementation('org.flywaydb:flyway-core')
 //	runtimeOnly('org.postgresql:postgresql')
 	compileOnly('org.projectlombok:lombok')
 	testImplementation('org.springframework.boot:spring-boot-starter-test')
+    testCompile('org.assertj:assertj-core:3.11.1')
+    testCompile('io.cucumber:cucumber-java8:4.0.0')
+    testCompile('io.cucumber:cucumber-junit:4.0.0')
+}
+
+configurations {
+    cucumberRuntime {
+        extendsFrom testRuntime
+    }
+}
+
+task cucumber() {
```

Comment:
Currently, this test won't run in CircleCI. I would make the test task depend(?) on this so that this test has to pass on CircleCI